GoogleCloudPlatform/datacatalog-connectors-hive
# ADD formatting strategy to entry fields that have hard limits on Data Catalog

**- What I did**

- Added `_format_id_with_hashing` logic to entry ID.
- Added `_format_display_name` logic to entry Display Name.
- Added `_format_linked_resource` logic to Linked Resource. 

**- How I did it**

Used the methods implemented in the commons library to make those fields compliant with Data Catalog hard limits.

**- How to verify it**
Run the hive connector on a hive server containing tables with long table names 200+, it should run successfully.

**- Description for the changelog**
Added formatting strategy to entry fields that have hard limits on Data Catalog.
Fixes #25.
